frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -12,11 +12,11 @@ import MUIDataTable, {
   CustomHeadLabelRenderOptions,
   MUIDataTableColumn,
 } from "mui-datatables";
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-import { ReviewResponse } from "../../../APIClients/ReviewAPIClient";
+import getReviews from "../../../APIClients/ReviewAPIClient";
```

Comment:
Can we change this import to:
`import reviewApiClient from "../../APIClients/ReviewAPIClient.ts";` so it's consistent with other files

In Login.tsx we import AuthApiClient.ts as follows:
`import authAPIClient from "../../APIClients/AuthAPIClient";` and then use it like `authApiClient.function()`